lib/resolvers/project.js

```diff
@@ -71,7 +71,7 @@ module.exports = ({ models }) => ({ action, data, id, meta }, transaction) => {
               issueDate: startDate.toISOString(),
               expiryDate,
               title: version.data.title,
-              licenceNumber: project.licenceNumber || generateLicenceNumber()
+              licenceNumber: project.licenceNumber || generateLicenceNumber('project', project.schemaVersion)
```

Comment:
The arguments passed to `generateLicenceNumber` looks _very_ different to those passed to a PIL above. That feels incorrect.

Reply:
```suggestion
              licenceNumber: project.licenceNumber || generateLicenceNumber(Project, transaction, 'project', project.schemaVersion)
```

Reply:
Actually, not that. The `generateLicenceNumber` call is async now, so needs to be pulled up into its own Promise step in the same as PILs.